jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -313,9 +313,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
   /* 1. */
-  /* This #def just gets around the syntax/style checker... */
-#define extended_property_ref_initialization { { 0 } , 0 }
-  ecma_extended_property_ref_t ext_property_ref = extended_property_ref_initialization;
+  ecma_extended_property_ref_t ext_property_ref = { .property_ref.value_p = NULL, .property_p = NULL };
```

Comment:
Yes, this is valid. Multiply open braces in the same line is what the vera rule cannot handle.